### DISCUSSION TITLE
Can it run on balanced mode on battery?

### DISCUSSION THREAD

**Discussion by ChucklesDroid** [AdnanHodzic/auto-cpufreq#632] [Q&A]
I tried the command `auto-cpufreq --force balanced` , after this I ran `auto-cpufreq --stats` which gave me the following result:
![image](https://github.com/AdnanHodzic/auto-cpufreq/assets/71250179/c9c275c6-df13-40e2-8420-c1d66d66b588)

I am kinda confused by the output is it running on powersave or performance governor

I am running a Fedora 38 Spin(KDE Plasma) and my laptop is lg gram 12th gen i7 1260p with 16gb ram

**Comment by AdnanHodzic** [comment]
_Setting to use: "**powersave**" governor._ 

I hope that answers your question, just as a reminder you can also use GUI instead of CLI if it could make some things clearer.
